### PR TITLE
Fix typo in setup.md

### PR DESCRIPTION
**Pull request by eramdam**
Not a big deal but it was bugging me :grin:

### CODE REVIEW

**Comment by MoOx** [comment]
Good catch. Thanks !

File: docs/content/setup.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ PostCSS runner that suits your workflow:
 
 ---
 
-## Wait? What were is previous cssnext interface?
+## Wait? What where is the previous cssnext interface?
```
Comment (jedmao) [inline docs/content/setup.md]: Isn't this still wrong? I think "What" should be removed.

Comment (Macxim) [inline docs/content/setup.md]: Definitely.
This should be:
`Wait, what? Where is the previous cssnext interface?`

Comment (jedmao) [inline docs/content/setup.md]: Agreed.

**Pull request merged**